addons/survey/static/tests/components/update_flag_field.test.js

```diff
@@ -0,0 +1,131 @@
+import { defineMailModels } from "@mail/../tests/mail_test_helpers";
+import { expect, test } from "@odoo/hoot";
+import { animationFrame, manuallyDispatchProgrammaticEvent, queryOne } from "@odoo/hoot-dom";
+import {
+    contains,
+    defineModels,
+    fields,
+    models,
+    mountView,
+    onRpc,
+} from "@web/../tests/web_test_helpers";
+
+class Survey extends models.Model {
+    question_and_page_ids = fields.One2many({ relation: "survey_question" });
+    session_speed_rating = fields.Boolean({ string: "Speed Reward" });
+    session_speed_rating_time_limit = fields.Integer({ string: "Speed Reward Time (s)" });
+
+    _onChanges = {
+        session_speed_rating: () => {},
+        session_speed_rating_time_limit: () => {},
+    };
```

Comment:
I prefer having the onchanges declared in each field, for clarity (_onChanges is better used when overriding onchanges in tests without redeclaring the field).
```suggestion
    session_speed_rating = fields.Boolean({ string: "Speed Reward", onChange: () => {} });
    session_speed_rating_time_limit = fields.Integer({ string: "Speed Reward Time (s)", onChange: () => {} });
```